xptifw/benchmark/object_table.cpp

```diff
@@ -0,0 +1,85 @@
+// #define XPTI_STATISTICS
+
+#include "helpers.hpp"
```

Comment:
Shouldn't we add LLVM header to all newly created files?

Reply:
Correct, I'll add one